src/library/Uno.Material/Styles/Controls/v2/AppBarButton.xaml

```diff
@@ -8,6 +8,17 @@
 					xmlns:ios="http://uno.ui/ios"
 					xmlns:uuc="using:Uno.UI.Controls"
 					mc:Ignorable="android ios">
+	<ResourceDictionary.ThemeDictionaries>
+		<ResourceDictionary x:Key="Light">
+			<x:Double x:Key="AppBarButtonHeight">64</x:Double>
```

Comment:
@kazo0 I have a little doubt, are we not missing documentation for AppBarButton like we have [here for Button](https://github.com/unoplatform/Uno.Themes/blob/master/doc/styles/Button.md) ?

Reply:
yeah it looks like we don't have a docs page for the styles/resources for AppBarButton. It's sort of a special case as these are used specifically for the NavigationBar. But I will add a docs page to the PR